doc/wg/core/notes/core-notes-2023-12-08.md

```diff
@@ -0,0 +1,79 @@
+# Tock Core Notes 2023-12-08
+
+## Attendees
+- Tyler Potyondy
+- Phil Levis
+- Andrew Imwalle
+- Alex Radovici
+- Alyssa Haroldson
+- Leon Schuermann
+- Johnathan Van Why
+- Hudson Ayers
+
+## Updates
+# Binary Working Group
+- Hudson: Rust binary size working group. Some people reached out over github. Johnathan discussed this last week.
+- Alyssa: Working to help make the documentation more understandable. 
+- Alyssa: Some interesting topics have come up already, in particular regarding string formatting.
+- Alyssa: I want to put together the argument that printf style strings are the smallest way to do this.
+- Hudson: I am interested in if the core library gets compiled with your optimizations even when you are not using build standard. It seems people have claimed this is the case, but I remember seeing some pretty different / worse codegen when I looked into this a year ago. This may not be up to date, but I would be curious what the current standing is. 
+- Alyssa: It makes sense to me that there would be a prebuilt copy optimized for size and a prebuilt copy optimized for space.
+- Alyssa: One other question, would inlining mirror require rebuilding?
+- Hudson: I do not think so. 
+- Alyssa: One other thought: unpacking large result objects often results in poor codegen even with inlining. Perhaps mirror inlining may help with this.
+- Hudson: I cannot remember if we turned this on in Tock.
+
+## TickV Discussion (https://github.com/tock/tock/discussions/3709)
+- Andrew: Just as a heads up, some of the agenda emails have been going to junk.
+- Andrew: I have been focusing on TickV and key value for my team.
+- Andrew: TickV spec states that a limitation is fragmentation. If something is written to a region and never erased, the flash will show as filled up even if they have been erased.
```

Comment:
The wording of this is a little misleading. I was referring to the situation where something is written to a region in flash memory and is never removed. In such cases, even if the rest of the region is cleared, the flash memory might still indicate that the region is full, despite having only one valid entry. The current implementation of garbage collect is unable to solve this problem, which leads to the flash filling up while only having a few valid entries.

Reply:
Went ahead and added this to the notes. Let me know if this is sufficient now and better captures your meaning.

Reply:
That looks great! Thank you!